indices/services.py

```
@@ -0,0 +1,53 @@
+        self.app_secret = os.getenv("KIS_APP_SECRET")
+        #self.base_url = "https://openapi.koreainvestment.com:9443" 실제 계좌용
+        self.base_url = "https://openapivts.koreainvestment.com:29443" # 모의계좌 (잠깐 사용)
+        self._token = None  # [추가] 토큰 저장용 변수
+
+    def get_token(self):
+        # 이미 토큰이 있으면 새로 받지 않고 그대로 반환
+        if self._token:
+            return self._token
+            
+        url = f"{self.base_url}/oauth2/tokenP"
+        res = requests.post(url, json={
```

Ruff (0.14.11)

Probable use of `requests` call without timeout

(S113)